cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -90,10 +103,7 @@ def _is_visium_and_is_single_true(self) -> bool:
         :rtype bool
         """
         if self.is_visium_and_is_single_true is None:
-            try:
-                self.is_visium_and_is_single_true = self._is_visium() and self.adata.uns["spatial"]["is_single"]
-            except AttributeError:
-                self.is_visium_and_is_single_true = False
+            self.is_visium_and_is_single_true = self._is_visium() and (self._is_single() or False)
```

Comment:
clarifying--why do we need or False here?

Reply:
My intention here is to ensure `is_visium_is_single_true` is a true boolean, and not just truthy/falsy. Specifically, the `or False` covers the case where `_is_single()` returns `None`. 

Do you feel that `bool(expression)` would be clearer here?

Reply:
gotcha. Yeah, I think it would be clearer with bool(expression) if you don't mind!